source/MaterialXView/Main.cpp

```diff
@@ -86,6 +89,9 @@ int main(int argc, char* const argv[])
     mx::HwSpecularEnvironmentMethod specularEnvironmentMethod = mx::SPECULAR_ENVIRONMENT_FIS;
     int envSampleCount = DEFAULT_ENV_SAMPLE_COUNT;
     float lightRotation = 0.0f;
+    int bakeWidth = -1;
```

Comment:
It looks like the existing pattern is to initialize these variables in the same order that they're documented above, so we should initialize "bakeWidth" just after "captureFilename".